packages/Telephony/Actions/SerialNumberInput.cs

```diff
@@ -48,10 +48,28 @@ public SerialNumberInput([CallerFilePath] string sourceFilePath = "", [CallerLin
         public BoolExpression AllowInterruptions { get; set; } = false;
 
         /// <summary>
-        /// Gets or sets the serial number pattern to use to decide when the dialog has aggregated the whole message.
+        /// Gets or sets if SerialNumberInput should accept alphabet characters. 
         /// </summary>
-        [JsonProperty("regexPattern")]
-        public StringExpression RegexPattern { get; set; }
+        /// <value>
+        /// Bool or expression which evalutes to bool.
+        /// </value>
+        [JsonProperty("acceptAlphabet")]
+        public BoolExpression AcceptAlphabet { get; set; }
+
+        /// <summary>
+        /// Gets or sets if SerialNumberInput should accept numbers. 
+        /// </summary>
+        /// <value>
+        /// Bool or expression which evalutes to bool.
+        /// </value>
+        [JsonProperty("acceptNumbers")]
+        public BoolExpression AcceptNumbers { get; set; }
```

Comment:
I think I'd very much like to remove both of these booleans and have `SerialNumberInput` just accept any alphanumeric inputs, aggregating the inputs until the desired batch length is met.

FYI @jameslew